src/components/Input/Input.js

```diff
@@ -16,20 +17,30 @@ export default class Input extends Component {
   focus = () => this.input.focus()
 
   render () {
-    const { style, ...restProps } = this.props
-
+    const { error, style, ...restProps } = this.props
+    let responseStyles = {}
+    if (error === true) {
+      responseStyles = styles.error
+    } else if (error === false) {
+      responseStyles = styles.success
+    }
     return (
       <TextInput
         {...restProps}
         style={[
           styles.input,
+          responseStyles,
           style,
         ]}
         ref={this.refInput}
-        placeholderTextColor='#9997b2'
+        placeholderTextColor={error ? 'red' : '#9997b2'}
```

Comment:
May I ask you to replace all color constants to variables from colors.js file?
If a color is missing there please add it.
Even colors like 'red' better to keep in variable there. For example: `errorColor: 'red'`